app/models/inventory.rb

```diff
@@ -2,23 +2,21 @@
 #
 # Table name: inventories
 #
-#  id               :integer          not null, primary key
-#  name             :string           not null
-#  deadline         :datetime         not null
-#  district_id      :integer          not null
-#  product_entry_id :integer
-#  data_entry_id    :integer
+#  id          :integer          not null, primary key
+#  name        :string           not null
+#  deadline    :datetime         not null
+#  district_id :integer          not null
 #
 
 class Inventory < ActiveRecord::Base
-  has_one :product_entry
-  has_one :data_entry
+  has_many :product_entries
+  has_many :data_entries
   belongs_to :district
 
-  accepts_nested_attributes_for :product_entry
-  accepts_nested_attributes_for :data_entry
+  accepts_nested_attributes_for :product_entries
+  accepts_nested_attributes_for :data_entries
 
   default_scope {
-    includes(:product_entry, :data_entry)
+    includes(:product_entries, :data_entries)
```

Comment:
mmm I'm not sure if we'd all all these entries to the default scope, if we're showing all the inventories in a list for the user to pick we don't necessarily need the product entries for that, this was fine before when we had 1-1 relationship but not that is 1-to-many it feels a little odd to me.